src/ngraph/op/fused/lstm_cell.cpp

```diff
@@ -225,12 +210,19 @@ NodeVector op::LSTMCell::decompose_op() const
     // --------------------
 
     Output<Node> X = input_value(0);
-    Output<Node> W = input_value(1);
-    Output<Node> R = input_value(2);
-    Output<Node> H_t = input_value(3);
-    Output<Node> C_t = input_value(4);
-    Output<Node> bias = get_bias();
-    NodeVector p_iof = get_peephole_weights();
+    Output<Node> H_t = input_value(1);
+    Output<Node> C_t = input_value(2);
+    Output<Node> W = input_value(3);
+    Output<Node> R = input_value(4);
+    Output<Node> bias = input_value(5);
+    NodeVector p_iof = builder::split(input_value(6), s_peepholes_count);
+
+    if (m_weights_format != op::LSTMWeightsFormat::IFCO)
```

Comment:
It is worth to add a comment why default format of weights is `op::LSTMWeightsFormat::IFCO` IMHO.

Reply:
Hmm this is already explained in the doc of `convert_node_format` function. But will add it here also.